src/azure-cli/azure/cli/command_modules/appservice/appservice_environment.py

```diff
@@ -350,8 +350,11 @@ def _get_unique_deployment_name(prefix):
     return prefix + random_string(16)
 
 
-def _build_ase_deployment_properties(name, location, subnet_id, virtual_ip_type,
+def _build_ase_deployment_properties(cli_ctx, name, location, subnet_id, virtual_ip_type,
                                      front_end_scale_factor=None, front_end_sku=None, tags=None):
+    from azure.cli.core.profiles import ResourceType, get_sdk
+    DeploymentProperties = get_sdk(cli_ctx, ResourceType.MGMT_RESOURCE_RESOURCES, 'DeploymentProperties', mod='models')
```

Comment:
same here as well

Reply:
After offline discussion with @panchagnula we decided to keep it this way.